API/api/controllers/event_controller.py

```
@@ -41,6 +41,17 @@ def create_event(request):
     except ValidationError as e:
         return Response(e.detail, status=status.HTTP_400_BAD_REQUEST)
     
+@api_view(["GET"])
+@permission_classes([IsAuthenticated])
+
+def get_events(request):
```

Ruff (0.15.12)

[warning] Unused function argument: `request`

(ARG001)